set_env_db_template.sh

```diff
@@ -0,0 +1,10 @@
+export DB_API_PORT=27017 #Port on the MongoDB server
```

Comment:
Add a shebang (that will make codacy also to be happy)

Reply:
Good, thanks for the advise, I didnt have to look into the codacy report